common/common.go

```diff
@@ -113,7 +114,9 @@ func zipAppDir(dirPath string) string {
 	}
 	defer zipFile.Close()
 
-	zip := archives.Zip{}
+	zip := archives.Zip{
+		Compression: zip.Deflate,
+	}
 	err = zip.Archive(ctx, zipFile, files);
```

Comment:
Sorry for commenting on the existing part.
The semicolon disappeared when I saved this file on my VSCode where [this extension](https://marketplace.visualstudio.com/items?itemName=golang.go) is installed.

I recommend deleting it if there is no reason to keep it.

```suggestion
	err = zip.Archive(ctx, zipFile, files)
```